CONTRIBUTING.md

```diff
@@ -1,3 +1,5 @@
 Contributing :)
 
+hahahaha
+
```

Comment:
Please use sentence case when writing prose, and don't forget punctuation.

Reply:
no